src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -205,4 +213,84 @@ else if (hourStart == hourEnd) {
     }
     return true;  
   }
+
+/**
+  Gets start and end times for display purposes -- converts times after 12 to standard
+  as opposed to military (i.e 13 to 1)
+*/
+  private String getStartTimeDisplay(String startTime, JSONObject json) {
+    String oldHourStart;
+    int hourStart;
+    String hourStartForDisplay;
+    String period;
+    int firstChar = Integer.parseInt(startTime.substring(0, 1));
+    int firstTwoChars = Integer.parseInt(startTime.substring(0, 2));
+    
+    if (firstChar == 0 && firstTwoChars != 00) {
+      hourStartForDisplay = startTime.substring(1, 2);
+      period = "am";
+    }
+    else if (firstTwoChars == 00) {
+      hourStartForDisplay = "12";
+      period = "am";
+    }
+    else if (firstTwoChars == 10 || firstTwoChars == 11) {
+      hourStartForDisplay = String.valueOf(firstTwoChars);
+      period = "am";
+    }
+    else if (firstTwoChars == 12) {
+      hourStartForDisplay = String.valueOf(firstTwoChars);
+      period = "pm";
+    }
+    else {
+      oldHourStart = startTime.substring(0, 2);
+      hourStart = Integer.parseInt(oldHourStart);
+      int hourStartForDisplayInt = hourStart - 12;
+      hourStartForDisplay = String.valueOf(hourStartForDisplayInt);
+      period = "pm";
+    }
+
+    String startMin = startTime.substring(3);
+    hourStartForDisplay += ":" + startMin + period;
+
+    return hourStartForDisplay;
+  }
+
+  private String getEndTimeDisplay(String endTime) {
+    String oldHourEnd;
```

Comment:
Can the getStartTimeDisplay and getEndTimeDisplay functions somehow be consolidated? It seems like it's mostly general repeated code.